stores/amazon.py

```diff
@@ -157,8 +161,25 @@ def something_in_stock(self):
             return False
 
     def get_captcha_help(self):
-        log.info("Stuck on a captcha. Please help.")
-        self.notification_handler.send_notification("Amazon bot is stuck on a captcha!")
+        try:
+            log.info("Stuck on a captcha... Lets try to solve it.")
+            captcha = AmazonCaptcha.from_webdriver(self.driver)
+            solution = captcha.solve()
+            log.info(f"The solution is: {solution}")
+            if solution is not "Not solved":
+                self.driver.find_element_by_xpath(
+                    '//*[@id="captchacharacters"]'
+                ).send_keys(solution + Keys.RETURN)
+            else:
+                self.driver.execute_script("window.location.reload()")
+                time.sleep(5)
+                self.get_captcha_help()
```

Comment:
Can we flip this if block around to avoid the double-negative "is not "not solved""? 

```suggestion
            if solution is "Not solved":
                self.driver.execute_script("window.location.reload()")
                time.sleep(5)
                self.get_captcha_help()
            else:
                self.driver.find_element_by_xpath(
                    '//*[@id="captchacharacters"]'
                ).send_keys(solution + Keys.RETURN)
```